shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -299,6 +304,7 @@ static bool DidFlagChange(const flutter::SemanticsNode& oldNode,
 
 void AccessibilityBridge::VisitObjectsRecursivelyAndRemove(SemanticsObject* object,
                                                            NSMutableArray<NSNumber*>* doomed_uids) {
+  [object accessibilityBridgeDidFinishUpdate];
```

Comment:
The name of this method doesn't seem to match to me with where it is being called.  If i had to name it I'd call it `accessibilityBridgeWillVisitObjectsRecursivelyAndRemove`, know what I mean?

Reply:
The accessibilityBridgeWillVisitObjectsRecursivelyAndRemove name is vague and is hard to interpret. I put it in VisitObjectsRecursivelyAndRemove because I don't want to do another tree walk just to call this method.

Reply:
The problem with what it is called now is that is leaking context information about where `VisitObjectsRecursivelyAndRemove` is called into calls higher in the call stack.  Would it make more sense to move that to where `VisitObjectsRecursivelyAndRemove` is being called?  Maybe you could make a method like this:
```
void finishUpdating(bridge, object) {
  doomed_uids = calculateDoomedUids();
  [object accessibilityBridgeDidFinishUpdate];
  bridge->VisitObjectsRecursivelyAndRemove(object, doomed_uids);
}
```

That would make the name make more sense.